sim_client/main.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import socket
+import time
+
+HOST = '127.0.0.1'  # The server's hostname or IP address
```

Comment:
Maybe test / allow 0.0.0.0 so another machine in the network could connect to the GC

Reply:
Oh thats the python script - nevermind